Framework/DataHandling/src/LoadHelper.cpp

```diff
@@ -353,62 +355,6 @@ void LoadHelper::recurseAndAddNexusFieldsToWsRun(NXhandle nxfileID, API::Run &ru
 
 } // recurseAndAddNexusFieldsToWsRun
 
-/**
```

Comment:
I guess you remove dumpNexusAttributes because it is unused in the code ?

Reply:
Indeed, it is not used anywhere in the codebase for quite a long time. There were parts of the code that depended on HDF4.